app/scripts/directives/logViewer.js

```diff
@@ -446,6 +446,10 @@ angular.module('openshiftConsole')
                   access_token: AuthService.UserStore().getToken()
                 });
 
+                var canI = $filter('canI');
+                var podsLogVersion = APIService.getPreferredVersion('pods/log');
+                var canViewOperationsLogs = canI(podsLogVersion, 'view');
```

Comment:
This is checking that you can view the log you're already viewing... @jcantrill Is this really what we want?

Reply:
This specific issue is that we changed the available index patterns if you are an operations user.  You used to get a list of every project in the cluster which was exceedingly long and we modified the seeding to provide only one which matched them all with 'project.*'.   The complementary change was not captured in the web console.  Operations users are being redirected to Kibana to use an index-pattern which does not exist.

Here we are trying to determine if the use is an operations user by using the same SAR we use in the multi-tenant plugin.

Reply:
Only operations users should be able to view pods/log from the logging project, so:
```
oc auth can-i view pods/log -n openshift-logging
```